### ISSUE TITLE
Concatenation

### ISSUE THREAD

**Issue by glenner003** [bazaarvoice/jolt#188]
Hi,
Im looking to use jolt in an integration layer. It looks a natural fit.
Is it possible to do concatenations.
We have the need the transform something like
```
{
  "property1": "foo",
  "property2": "bar"
}
```
into
```
{
  "newProperty": "STATIC_TEXT_foo_bar"
}
```
Or do I need to foresee a custom bean the implement this?

Regards,

Glenn

**Comment by milosimpson** [comment]
A custom bean, I can do it in 3 Shifts, but its ugly / a total hack.

**Comment by mhubbard** [comment]
Are there any plans to implement concatenation? I've been using jolt to integrate with a client but need to shift firstName and lastName fields into a fullName field.

When you say you can hack it with 3 shifts do you mean 3 different tranformations each with their own spec?

**Comment by milosimpson** [comment]
Yes, it is actually in progress, see https://github.com/bazaarvoice/jolt/pull/222 .   Note we are specifically marking these new "Transforms" as "beta" as we are still working on them, and "reserve" the right to make backwards incompatible changes to them.

For context the new transform you would use would be "default-beta", and the spec would look like
```
{
    "newProperty": "=concat(STATIC_TEXT_,@(1,property1),_,@(1,property2))"
}
```

**Comment by milosimpson** [comment]
This is now available in Jolt 0.0.22 with the new Modify transform.
